src/Admin/AbstractAdmin.php

```diff
@@ -784,6 +784,8 @@ public function buildDatagrid()
      * Returns the name of the parent related field, so the field can be use to set the default
      * value (ie the parent object) or to filter the object.
      *
+     * NEXT_MAJOR: Remove this exception
```

Comment:
Some time ago we started adding hassers in order to avoid calling getters on invalid scenarios. Shouldn't we add `hasParentAssociationMapping()` here?

Reply:
Looking at the comment, code will be in next major
```
if ($this->isChild()) {
    $parent = $this->getParent()->getCode();
    
    return $this->parentAssociationMapping[$parent];
}

return null;
```

Should we throw an exception if the admin is not a child instead ? (No need for an hasser, the dev can use `isChild`).

Should we have an hasser, hasParentAssociationMapping to check
```
if (\array_key_exists($parent, $this->parentAssociationMapping)) {
```
?